### DISCUSSION TITLE
Max Charge & Discharge Current?

### DISCUSSION THREAD

**Discussion by muhkuh666** [nathanmarlor/foxess_modbus#515] [Q&A] [answered]
Hello, where can I find the max allowed currents? I have a H3 6kW inverter!

**Comment by canton7** [comment]
Please use discussions for general questions. 

![image](https://github.com/nathanmarlor/foxess_modbus/assets/568104/ce72fa07-8a05-4e30-9bc9-3291a95e7f98)

**Comment by canton7** [comment, marked as answer]
I don't think the max currents are exposed anywhere by the inverter. You should be fine to leave these at their default values, unless you specifically want to decrease them.

There's no harm in having them set "too high" -- the current actually supplied to/from the batteries is the max of the Max Charge/Discharge current, and the current requested by the BMS, so we'll never give the batteries more current than the BMS wants to have.

**Reply by muhkuh666** [reply]
Thank you, I found something in the manual
![1111](https://github.com/nathanmarlor/foxess_modbus/assets/127597208/1c32aafb-56a8-49ca-8059-676c53f5e645)
